### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by tinkerbell00**
Dashboard design 
Charts position;
Charts dimensions;
Edit dashboard/components menu;
Share dashboard page;

### CODE REVIEW

**Comment by tinkerbell00** [comment]
[Dashboard](https://trello.com/c/bG0fWEbg/15-dashboard)

**Comment by qdequippe** [comment]
I have an Exception when I try to access / (http://127.0.0.1:8000/)

```
An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "app_add_component" as such route does not exist.").
```
Do you have the same exception?

Yes, it's the path to another page, if i comment the line like this {# path('app_add_component') #} it works

File: src/AppBundle/Resources/views/App/dashboard.html.twig [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+                                                    aria-hidden="true"></i>
+                                            Line</a></li>
+                                    <li><a href="javascript:void(0)" onclick="change_chart_type('column', {{ key }})"><i
+                                                    class="fa fa-bar-chart" aria-hidden="true"></i> Column</a></li>
+                                    <li><a href="javascript:void(0)" onclick="change_chart_type('area', {{ key }})"><i
+                                                    class="fa fa-area-chart" aria-hidden="true"></i> Area</a></li>
+                                    <li><a href="javascript:void(0)" onclick="change_chart_type('pie', {{ key }})"><i
+                                                    class="fa fa-pie-chart" aria-hidden="true"></i> Pie</a></li>
+                                    <li><a href="javascript:void(0)" onclick="change_chart_type('value', {{ key }})"><i
+                                                    class="fa fa-usd"
```
Comment (qdequippe) [inline src/AppBundle/Resources/views/App/dashboard.html.twig]: The dropdown goes out of the page

**Review by qdequippe — CHANGES_REQUESTED** [review]
`web/assets/vendor` to add to .gitignore

File: src/AppBundle/Resources/views/App/dashboard.html.twig (hunk 1 above)
Comment (tinkerbell00) [reply src/AppBundle/Resources/views/App/dashboard.html.twig]: Dropdown fixed

**Comment by qdequippe** [comment]
too many files in this PR, maybe check gitignore file

**Review by qdequippe — APPROVED** [review]

**Pull request merged**